en/docs/setup/getting-started-with-the-management-console.md

```diff
@@ -66,102 +63,94 @@ divided into different sections.
 
-**Related links**
+!!! info "Related links" 
+    See the following topics for instructions on how to use each of these
+    menu items. 
 
-See the following topics for instructions on how to use each of these
-menu items.
-
--   See [Configuring the Policy Administration
-    Point](../../tutorials/configuring-the-policy-administration-point) for
-    configuring the policy administration point.
--   See [Configuring the Policy Decision
-    Point](Configuring_the_Policy_Decision_Point) for configuring the
-    policy decision point.
+    -   See [Configuring the Policy Administration
+        Point](../../tutorials/configuring-the-policy-administration-point) for
+        configuring the policy administration point.
+    -   See [Configuring the Policy Decision
+        Point](../../tutorials/configuring-the-policy-decision-point) for configuring the
+        policy decision point.
 
 #### Manage section
 
-![](attachments/103328985/103328986.png)
-
-**Related links**
-
-See the following topics for instructions on how to use each of these
-menu items.
-
--   See [Workflow Management](../../learn/workflow-management) for information on
-    working with workflows.
--   See [Managing Challenge Questions](Managing_Challenge_Questions)
-    for information on configuring challenge questions. Also see [Using
-    the End User Dashboard](../../learn/using-the-end-user-dashboard) for
-    information on challenge questions.
--   See [Email Templates](Email_Templates) for configuring email
-    templates. Also see [Customizing Automated
-    Emails](https://docs.wso2.com/display/IS530/Customizing+Automated+Emails)
-    for information on how to customize these templates.
--   See [Creating New Keystores in the WSO2 Administration
-    Guide](https://docs.wso2.com/display/ADMIN44x/Creating+New+Keystores)
-    for configuring keystores.
+![Manage menu](../../assets/img/setup/manage-menu-png)
+
+!!! info "Related links"  
+    See the following topics for instructions on how to use each of these
+    menu items.
+
+    -   See [Workflow Management](../../using-wso2-identity-server/workflow-management) for information on
+        working with workflows.
+    -   See [Managing Challenge Questions](../../using-wso2-identity-server/managing-challenge-questions)
+        for information on configuring challenge questions. Also see [Using
+        the End User Dashboard](../../using-wso2-identity-server/using-the-end-user-dashboard) for
+        information on challenge questions.
+    -   See [Email Templates](../../using-wso2-identity-server/email-templates) for configuring email
+        templates. Also see [Customizing Automated
+        Emails](../../using-wso2-identity-server/customizing-automated-emails)
+        for information on how to customize these templates.
+    -   See [Creating New Keystores in the WSO2 Administration
+        Guide](../../admin-guide/creating-new-keystores)
+        for configuring keystores.
```

Comment:
change the link paths according to the new folder structure

Reply:
Fixed